src/components/Items/Header.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div :class="containerClass">
+    <!-- skip button -->
+    <icon-button
+      v-if="isSkipEnabled"
+      :titleConfig="skipButtonTitleConfig"
+      @click="skipItem"
+      :buttonClass="skipButtonClass"
+      data-test="skip"
+    ></icon-button>
+  </div>
+</template>
+
+<script>
+import IconButton from '@/components/UI/Buttons/IconButton.vue'
+
+export default {
+  name: 'Header',
+  data() {
+    return {
+      // styling class for the skip button
+      skipButtonClass:
+        'bg-primary hover:bg-primary-hover p-1 pl-4 pr-4 sm:p-2 sm:pl-10 sm:pr-10 lg:p-4 lg:pl-10 lg:pr-10 rounded-md shadow-xl disabled:opacity-50 disabled:pointer-events-none h-full'
```

Comment:
Why using data properties for setting classes rather than using the `@apply` tag inside CSS. I kind of prefer `@apply` just as personal opinion